API/src/main/java/de/brandwatch/minianalytics/api/controller/QueryController.java

```diff
@@ -33,6 +33,7 @@ public Query createQuery(@RequestParam String query) {
             logger.info("POST /queries: {}", query);
             return queryService.createQuery(query);
         } catch (Exception e) {
+            logger.error(e.getMessage());
```

Comment:
This is not best practice. The  logger knows how to handle exceptions when you pass `logger.error("Some message", e)`